docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+name: expensetrackergrupotwo
+
+services:
+  webapp:
+    container_name: expense-tracker-api
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports: 
+      - "5001:5001" 
+    environment:
+      - ASPNETCORE_ENVIRONMENT=Development
+      - Database__ConnectionString__DefaultConnection=Host=postgres;Port=5432;Database=ExpenseTrackerGrupo2;Username=expenseTrackerGrupoTwo;Password=123456789
```

Comment:
Would you mind to change the name of the Database please? It's not common with upper letters.